tests/constants.py

```diff
@@ -49,3 +50,101 @@
 COMMAND = "eporeporjgpeorjgpeorjgpeorjgpeorjgpeorjg"
 
 IMPORTED_IMAGE_ID = 'eee28534d167d7b3297eace1fc32c46aabedc40696e48ae04c7654f974700cc2'
+
+REACTOR_CONFIG_MAP = dedent("""\
+koji:
+    hub_url: https://koji.example.com/hub
+    root_url: https://koji.example.com/root
+    auth:
+        ssl_certs_dir: /var/run/secrets/atomic-reactor/kojisecret
```

Comment:
`ssl_certs_dir` and `krb_*` should not be allowed together.

Reply:
yeah fixed after fixed schema